src/Stratis.Features.Diagnostic/Controllers/Models/PeerStatisticsModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Stratis.Features.Diagnostic.PeerDiagnostic;
+
+namespace Stratis.Features.Diagnostic.Controllers.Models
+{
+    public class PeerStatisticsModel
+    {
+        public string PeerEndPoint { get; set; }
```

Comment:
@MithrilMan I'm just being pedantic but can you order these fields alphabetically?

Reply:
Also they should all be `readonly`?

Reply:
it's equivalent, they have the getter only